src/osemosys/schemas/model.py

```diff
@@ -38,14 +36,17 @@ class RunSpec(OSeMOSYSBase):
     #transmission_technologies: List[TechnologyTransmission]
 
     # reserve margins if any
-    reserve_margins_commodity: List[Dict[str,RegionYearData]] | None
-    reserve_margins_technology: List[Dict[str, RegionYearData]] | None
+    reserve_margins_level: RegionYearData | None
+    reserve_margins_commodity: RegionCommodityYearData | None
+    reserve_margins_technology: RegionTechnologyYearData | None
 
     # renewable targets
     renewable_targets: RegionYearData | None
     
+    # Default values
+    #TODO
 
-    def to_simplicity(self, comparison_directory) -> Dict[str,str]:
+    def to_csv(self, comparison_directory) -> Dict[str,str]:
```

Comment:
this isn't just "to csv" right? this is to the very specific format dictated by otool? perhaps it should be "to[from]_otool"?